### PR TITLE
Feature/dev 11910

### PR DESCRIPTION
**Pull request by lordpixel**
This PR, if merged, will add support for Semantic UI React in cosmos.

### CODE REVIEW

File: design/ui/pagination/cosmos.decorator.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+// The only thing this decorator will do for now is import our SUI custom CSS.
+// There are currently issues with the out of the box compiled styles there
+// and possibly some missing loaders so we will use a different variant for now
+// that does not have any missing font/image dependencies.
+
+// import 'semantic-ui-css-custom';
```
Comment (brianespinosa) [inline design/ui/pagination/cosmos.decorator.js:6]: Uncomment this and see that we are getting the new CSS in for all cosmos fixtures

File: design/ui/pagination/rollup.config.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+      use: ['sass'],
+    }),
+    babel({
+      babelrc: false,
+      babelHelpers: 'runtime',
+      presets: ['espressive'],
+      plugins: ['@babel/plugin-transform-runtime'],
+    }),
+  ],
+};
```
Comment (brianespinosa) [inline design/ui/pagination/rollup.config.js:41]: Please double check this is the same with button

File: design/ui/semantic-ui-css-custom/src/fonts/inter/LICENSE.txt [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+DISCLAIMER
+THE FONT SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO ANY WARRANTIES OF
+MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT
+OF COPYRIGHT, PATENT, TRADEMARK, OR OTHER RIGHT. IN NO EVENT SHALL THE
+COPYRIGHT HOLDER BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY,
+INCLUDING ANY GENERAL, SPECIAL, INDIRECT, INCIDENTAL, OR CONSEQUENTIAL
+DAMAGES, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+FROM, OUT OF THE USE OR INABILITY TO USE THE FONT SOFTWARE OR FROM
+OTHER DEALINGS IN THE FONT SOFTWARE.
```
Comment (brianespinosa) [inline design/ui/semantic-ui-css-custom/src/fonts/inter/LICENSE.txt:92]: Remove all Inter for now

File: design/ui/pagination/package.json [hunk 4]
```diff
@@ -0,0 +1,45 @@
+{
+  "name": "@espressive/pagination",
+  "version": "1.0.0",
+  "main": "dist/index.js",
+  "module": "es/index.js",
+  "scripts": {
+    "rollup": "rm -rf dist && rm -rf es && BABEL_ENV=production rollup -c",
+    "rollup:watch": "rm -rf dist && rm -rf es && BABEL_ENV=production rollup -c --watch"
```
Comment (brianespinosa) [inline design/ui/pagination/package.json:8]: This needs to have development env for rollup

**Comment by brianespinosa** [comment]
@lordpixel looks like the snapshot for this PR is still failing. Please press the button requesting another review once that build passes and I will get this merged. <3

**Comment by lordpixel** [comment]
@brianespinosa, fixing the unit tests would create a lot of conflicts with DEV-11892 so I went ahead and merged both, hence closing this one.

**Pull request closed without merging**